CONTRIBUTING.md

```diff
@@ -81,6 +81,8 @@ For this tutorial we will take the word «Obsessive Compulsive Disorder», as it
 
 We have created some examples that you can use as the baseline for your work. Take a look at the [File Examples](https://www.selfdefined.app/documentation/examples/) section of our documentation.
 
+**Related:** [When should I open an issue versus a pull request (PR)?](https://www.selfdefined.app/documentation/frequently-asked-questions/#when-should-i-open-an-issue-versus-a-pull-request-(pr)%13F)
+
 ## Get Help
 
 If you've already tried reading through our [documentation](https://www.selfdefined.app/documentation/) and are stuck, we're here to help and ask your questions:
```

Comment:
Please add a link to [join our Slack community](https://join.slack.com/t/selfdefined/shared_invite/zt-fczgm8b6-8ZZgHvLutNDXo~NjwaL7Iw) under the **Get Help** section: